dashboard/lib/services/user/password_resetter_by_email.rb

```diff
@@ -1,9 +1,9 @@
 module Services
   module User
-    class PasswordResetter < Services::Base
+    class PasswordResetterByEmail < Services::Base
       attr_reader :email
 
-      def initialize(email:)
+      def initialize(email: nil)
```

Comment:
A default value makes the argument optional, but it should be required in this service, so please use:
```suggestion
      def initialize(email:)
```
or just
```suggestion
      def initialize(email)
```